app/dashboard/page.tsx

```diff
@@ -3,12 +3,23 @@ import React, { FC, useEffect, useState } from "react";
 import Navbar from "../components/Header";
 import accountHelper from "../helpers/account";
 import Footer from "../components/Footer";
+import { IPaymentData } from "../interfaces/payments";
+import { sendPayment } from "../helpers/payments";
+import PaymentModal from "../components/PaymentModal";
+import { IFormErrors } from "../interfaces/errors";
+import paymentFormValidation from "../helpers/paymentFormValidation";
 
 const Dashboard: FC = () => {
   const [isFunded, setIsFunded] = useState(true);
   const [publicKey, setPublicKey] = useState("" as string);
   const [isLogged, setIsLogged] = useState(true);
   const [balance, setBalance] = useState("" as string | undefined);
+  const [paymentData, setPaymentData] = useState({} as IPaymentData);
```

Comment:
Avoid using the word data, everything's data.

Reply:
This constant store the I have changed the name paymentData for paymentSummary